src/vs/workbench/contrib/terminalContrib/accessibility/browser/terminalAccessibleBufferProvider.ts

```diff
@@ -50,6 +51,13 @@ export class TerminalAccessibleBufferProvider extends DisposableStore implements
 		}));
 	}
 
+	onKeyDown(e: IKeyboardEvent): void {
+		if (!this._instance.shouldProcessKeyEvent(e.browserEvent) || !isSingleLetterKey(e.browserEvent)) {
+			return;
+		}
+		this._instance.focus();
+	}
```

Comment:
Does this work by `keyup` getting handled by xterm.js?

Reply:
Yes